plugins/Jobs.cpp

```diff
@@ -193,6 +194,57 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
         return true;
     }
 
+    // ----------------------------------------------------------------------
+    else if (SIEquals(request.methodLine, "CancelJob")) {
+        // - CancelJob(jobID)
+        //
+        //     Cancel a QUEUED, RUNQUEUED, FAILED job from a sibling.
+        //     (actually, it allows you to cancel non siblings, but you
```

Comment:
Ok!! I get asked about adding more context all the time, so now I tend to overexplain things lol